config-devel.toml

```diff
@@ -25,8 +25,8 @@ content_endpoint = "/api/v1/content"
 insights_advisor_url = "https://console.redhat.com/openshift/insights/advisor/clusters/{cluster_id}"
 cluster_details_uri = "https://console.redhat.com/openshift/details/{cluster_id}#insights"
 rule_details_uri = "https://console.redhat.com/openshift/details/{cluster_id}/insights/{module}/{error_key}"
-# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-cooldown = "1m"
+# valid units are SQL epoch time units: months days hours minutes seconds"
+cooldown = "1 minutes"
```

Comment:
is the plural needed there?

Reply:
not needed. The unit can be any of `millennium, century, decade, year, month, week, day, hour, minute, second, millisecond, microsecond`; or the abbreviation (y, m, d, etc.,), or the plural forms (months, days, etc.).

But I think plural form, and "restricting" it from months to seconds makes sense when defining the cooldown =)

Reply:
ok